pages/internship.js

```diff
@@ -0,0 +1,116 @@
+import Link from 'next/link';
+import React from 'react';
+
+import Faq from 'react-faq-component';
+import Carousel from 'react-multi-carousel';
+
+import Banner from '../components/Banner';
+import CommitteeInternshipInfo from '../components/Internship/CommitteeInternshipInfo';
+import InternshipTimeline from '../components/Internship/InternshipTimeline';
+import NextSteps from '../components/Internship/NextSteps';
+import Layout from '../components/Layout';
+
+import data from '../data';
+import 'react-multi-carousel/lib/styles.css';
+
+function InternshipPage(){
+    const { committees, internship } = data;
+    const { items, testimonials, QA, FAQStyles, responsive } = internship;
+    return(
+        <Layout>
+            <Banner decorative />
+            <div className="content-section text-center">
+                <h1>Internships</h1>
+                What do you do as an intern? Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+                sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Tristique sollicitudin
+                nibh sit amet commodo nulla. Amet facilisis magna etiam tempor.
+            </div>
+            <CommitteeInternshipInfo
+                committees={committees}
+            />
+            <div className="content-section text-center">
+                <h1>Applications</h1>
+                <InternshipTimeline items={items}/>
+                <div className="application-information">
+                    <div>
+                        {items.map((item, i) => (
+                        <div key={i} className='application-item'>
+                            {item.date} &nbsp;
+                            {item.name}
+                            <p>{item.info}</p>
+                        </div>
+                        ))}
+                        -- more dates will be added throughout the quarter --
+                    </div>
+                    <div className='application-info-card'>
+                        <h3>Internship Applications Reopen Fall 2021</h3>
+                        <div className='application-button'>
+                            <Link href="">
```

Comment:
We shouldn't have blank hrefs; at the very least, should mark this with a TODO!

Reply:
Going to unresolve, since we're merging this before the application comes out.

Reply:
would it make sense to redirect to a page saying "coming soon"? I'll ask Ava how she prefers to handle this as well.

Reply:
talked to ava - hiding the button until apps are out!